cmd/edit/machinepool/nodepool.go

```diff
@@ -24,14 +24,16 @@ func editNodePool(cmd *cobra.Command, nodePoolID string, clusterKey string, clus
 	isAutoscalingSet := cmd.Flags().Changed("enable-autoscaling")
 	isLabelsSet := cmd.Flags().Changed("labels")
 	isTaintsSet := cmd.Flags().Changed("taints")
-	isLabelOrTaintSet := isLabelsSet || isTaintsSet
 	isVersionSet := cmd.Flags().Changed("version")
 	isAutorepairSet := cmd.Flags().Changed("autorepair")
 	isTuningsConfigSet := cmd.Flags().Changed("tuning-configs")
 
+	// isAnyAdditionalParameterSet is true if at least one parameter not related to replicas and autoscaling is set
+	isAnyAdditionalParameterSet := isLabelsSet || isTaintsSet || isVersionSet || isAutorepairSet || isTuningsConfigSet
+
 	// if no value set enter interactive mode
 	if !(isMinReplicasSet || isMaxReplicasSet || isReplicasSet || isAutoscalingSet || isLabelsSet || isTaintsSet ||
-		isAutorepairSet || isTuningsConfigSet) {
+		isVersionSet || isAutorepairSet || isTuningsConfigSet) {
```

Comment:
Can we substitute this for `isAnyAdditionalParameterSet`?

Reply:
good idea, changed that